XenModel/XenAPI/VM.cs

```diff
@@ -3590,7 +3590,7 @@ public static void set_protection_policy(Session session, string _vm, string _va
         /// <param name="_value">The value</param>
         public static void set_snapshot_schedule(Session session, string _vm, string _value)
         {
-            session.proxy.vm_set_snapshot_schedule(session.uuid, (_vm != null) ? _vm : "", (_value != null) ? _value : "").parse();
+            session.proxy.vm_set_snapshot_schedule(session.uuid, (_vm != null) ? _vm : "", (_value != null) ? _value : Helper.NullOpaqueRef).parse();

```

Comment:
To be explicit, if this file is changed, the changes will be reverted whenever we next update it from the autogenerated one.

In any case, I'm confused about this fix. Why does VMPP not need it? Does the API behave differently for VMSS and VMPP? If so, the VMSS API should be fixed instead.